src/topp/FeatureFinderCentroided.cpp

```diff
@@ -42,6 +42,9 @@
 #include <OpenMS/FORMAT/MzQuantMLFile.h>
 #include <OpenMS/METADATA/MSQuantifications.h>
 #include <OpenMS/SYSTEM/File.h>
+#include <OpenMS/FILTERING/TRANSFORMERS/ThresholdMower.h>
```

Comment:
this could be removed

Reply:
k will do in another PR - thanks

Reply:
Changed that on my fork. I don't know if I need to open another PR or if you can just reopen this one

Reply:
Thanks but I already pushed it to develop. So I guess its fine